### ISSUE TITLE
ignore files

### ISSUE THREAD

**Issue by karneaud** [fizker/minifier#16]
can you ignore files?

**Comment by fizker** [comment]
No. What would be the point? All files going over the wire should be minified.

**Comment by karneaud** [comment]
some file within a dependency package are sometimes not used which is usually less than the majority. Rather than list out each file you want to use we can "glob" match files and exclude those we do not want.... 


shame though

**Comment by fizker** [comment]
How about filtering the list before passing it in? Then we would have one less filter-implementation to struggle with

**Comment by karneaud** [comment]
using the CLI?

**Comment by fizker** [comment]
Sure. I don't know how you use it now, but [xargs](http://linuxcommand.org/man_pages/xargs1.html) can probably help you pass in a list as individually files. Or you could enumerate the list in bash or another script-language.
